pkgs/by-name/ru/ruri/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  libcap,
+  libseccomp,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ruri";
+  version = "3.8";
+
+  src = fetchFromGitHub {
+    owner = "Moe-hacker";
+    repo = "ruri";
+    rev = "v${version}";
+    fetchSubmodules = false;
+    sha256 = "sha256-gf+WJPGeLbMntBk8ryTSsV9L4J3N4Goh9eWBIBj5FA4=";
+  };
+
+  buildInputs = [
+    libcap
+    libseccomp
+  ];
+
+  enableParallelBuilding = true;
```

Comment:
I don't see much files to compile, and this make it possible to be not reproducible, so it may be better to just remove it.
```suggestion
```